src/parseDicom.js

```diff
@@ -32,7 +32,7 @@ export default function parseDicom (byteArray, options) {
 
     const transferSyntaxElement = metaHeaderDataSet.elements.x00020010;
 
-    return byteArrayParser.readFixedString(byteArray, transferSyntaxElement.dataOffset, transferSyntaxElement.length);
+    return transferSyntaxElement.Value || byteArrayParser.readFixedString(byteArray, transferSyntaxElement.dataOffset, transferSyntaxElement.length);
```

Comment:
is there any chance to have transferSyntaxElement.Value === [] (by mistake) but there is actually a transferSyntax.

This scenario can occur because 
....
const metaHeaderDataSet = {
        elements: {x00020010: { tag: 'x00020010', vr: 'UI', Value: options.TransferSyntaxUID }},
        warnings, 
      };
      ....
      If you see above Value comes from options which is external and there is no validation

Reply:
I added documentation.  No, there isn't validation here.  I could check for an array, and get the zero'th element if you think that is worthwhile.